app/views/game_masters/show.html.erb

```diff
@@ -1,22 +1,79 @@
 <p id="notice"><%= notice %></p>
 
-<p>
-  <strong>Table:</strong>
-  <%= @game_master.table.session.name %> <%= @game_master.table.scenario.name %>
-</p>
+<!-- breadcrumbs -->
+<ol class="breadcrumb">
+  <li><%= link_to "#{@event.name}", event_path(@event) %></li>
+  <li><%= link_to "#{@session.long_name}", event_session_path(@event, @session) %></li>
+  <li><%= link_to "#{@table.long_name}", event_session_table_path(@event, @session, @table) %></li>
+  <li>Game Master Information</li>
+</ol>
 
-<p>
-  <strong>GM:</strong>
-  <%= @game_master.user_event.user.name %>
-</p>
 
+<div class="page-header">
+  <h1><%= @event.name %></h1>
+  <h2><%= @session.name %>
+    <small><%= @session.timeslot %></small>
+  </h2>
+  <h3><%= @table.long_name %></h3>
+</div>
 
-<p>
-  <strong>Table Number:</strong>
-  <%= @game_master.table_number %>
-</p>
+<% player = @game_master&.user_event&.user %>
+<%#
+table info
+table number
+session ?
+%>
+<h4>Table Information</h4>
+<div class="panel panel-padded" style="border: solid thin black;">
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Table Location:</strong></div>
+    <div class="col-md-6 col-sm-5"><%= @table.location %></div>
+  </div>
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Table Information:</strong></div>
+    <div class="col-md-6 col-sm-5">Put stuff here</div>
+  </div>
+</div>
 
+<h4>Game Master's Information</h4>
+<div class="panel panel-padded" style="border: solid thin black;">
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Game Master:</strong></div>
+    <div class="col-md-6 col-sm-5"><%= player.formal_name %></div>
+  </div>
+  <div class="row">
+    <div class="col-md-2 col-sm-3"><strong>Email:</strong></div>
+    <div class="col-md-6 col-sm-5"><%= player.email %></div>
```

Comment:
Should the game master be able to see the player's email? If it's to send them messages about the game, it'd probably be better to give them a mailing list for the game where they can't see the player's email directly. (Could we even make a mailing list per game? Is that a power GM's should even have?) If GMs need to communicate with players, would just having the game server do?

Reply:
Probably not... this page is going to be updated with the next set of changes.